libs/common/types.hpp

```diff
@@ -303,5 +304,8 @@ namespace iroha {
     return typeid(Base) == typeid(ptr);
   }
 
+  // Peer address
+  using AddressType = std::string;
```

Comment:
It does not seem necessary, since old model can use shared model.

Reply:
mb it worth to keep it both in the old model and in the shared model?